<launch>
  <node name="neo_node_pc2"	pkg="neo_ros_pc2"  type="neo_node_pc2" output="screen">
  	<param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
  	<param name="serial_baudrate"     type="int"    value="115200"/>
  	<param name="frame_id"            type="string" value="laser_frame"/>
  	<param name="rotation_speed"	  type="int" value="5"/>
  </node>
</launch>
